adserver/management/commands/payouts.py

```diff
@@ -213,7 +225,7 @@ def handle(self, *args, **kwargs):
                     "to": [user.email for user in publisher.user_set.all()],
                     "sender_name": "EthicalAds by Read the Docs",
                     "subject": f"EthicalAds Payout - {publisher.name}",
-                    "options": {"archive": False},
+                    "options": {"archive": True},
```

Comment:
Is this an inadvertent change?

Reply:
Nope -- it was spamming my inbox, so I'm archiving the messages by default.